transforms/code/repo_level_ordering/ray/test/test_internal_wrappers.py

```diff
@@ -0,0 +1,129 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import pyarrow as pa
+from data_processing.test_support.transform.table_transform_test import (
+    AbstractTableTransformTest,
+)
+from data_processing.transform import AbstractTableTransform
+from data_processing.utils import get_logger
+from dpk_repo_level_order.internal.repo_level_wrappers import (
+    get_dominant_language_func,
+    get_sorting_func,
+)
+
+
+class PassThroughTransform(AbstractTableTransform):
```

Comment:
test_support package in the main python lib includes a NoopTransform.  If this meets your needs, may be use it instead.

Reply:
nice to have, but not mandatory